### PR TITLE
Add option for auth in config

### PR DESCRIPTION
**Pull request by vishesh92**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -21,7 +21,8 @@ openvpn_log: /var/log/openvpn.log                   # Log's directory
 openvpn_keepalive: "10 120"
 openvpn_ifconfig_pool_persist: ipp.txt
 openvpn_comp_lzo: yes                               # Enable compression
-openvpn_cipher: AES-256-GCM                         # Encryption algorithm
+openvpn_cipher: AES-256-CBC                         # Encryption algorithm
```
Comment (manjitkumar) [inline defaults/main.yml:24]: Why did we change this encryption algorithm?

Comment (vishesh92) [reply defaults/main.yml:24]: GCM is not supported in some versions.

Comment (manjitkumar) [reply defaults/main.yml:24]: okay

**Comment by manjitkumar** [comment]
LGTM

**Pull request merged**